tests/test_cli.py

```diff
@@ -98,3 +98,76 @@ def test_end_to_end_multiple(capsys, tmp_path):
   assert not cap.out
   assert "foo.txt" in cap.err
   assert "Key[827bc3b2:EdPK] Signature verified" in cap.err
+
+
+def test_end_to_end_shortargs_armored(capsys, tmp_path):
+  from covert.__main__ import main
+  import sys
+  fname = tmp_path / "crypto.covert"
+
+  # Encrypt foo.txt into crypto.covert
+  sys.argv = "covert -eRao tests/keys/ssh_ed25519.pub".split() + [ str(fname) ] + [ 'tests/data/foo.txt' ]
+  ret = main()
+  cap = capsys.readouterr()
+  assert not ret
+  assert not cap.out
+  assert "foo" in cap.err
+
+  # Decrypt with key
+  sys.argv = "covert -di tests/keys/ssh_ed25519".split() + [ str(fname) ]
+  ret = main()
+  cap = capsys.readouterr()
+  assert not ret
+  assert not cap.out
+  assert "foo.txt" in cap.err
+
+
+def test_end_to_end_armormaxsize(capsys, tmp_path):
+  from covert.__main__ import main
+  import sys
+  fname = tmp_path / "test.dat"
+  outfname = tmp_path / "crypto.covert"
+
+  # Write 31 MiB on test.dat
+  f = open(f"{str(fname)}", "wb")
+  f.seek(32505855)
+  f.write(b"\0")
+  f.close()
+
+  # Encrypt test.dat with armor and padding
+  sys.argv = f"covert -e --password verytestysecret --pad 0 -ao".split() + [ str(outfname), str(fname) ]
+  ret = main()
+  cap = capsys.readouterr()
+  assert not ret
+  assert not cap.out
+
+  # Decrypt crypto.covert with passphrase
+  sys.argv = "covert -d --password verytestysecret".split() + [ str(outfname) ]
+  ret = main()
+  cap = capsys.readouterr()
+  assert not ret
+  assert not cap.out
+
+
+
+  # Write file with size too large for --armor
+  f = open(f"{str(fname)}", "wb")
+  f.seek(42505855)
+  f.write(b"\0")
+  f.close()
```

Comment:
Whenever possible, use
```python
with open(...) as f:
   # do something with f but no need to f.close() because leaving the with-block closes automatically
```

Reply:
Also no reason to use str() in f-string, it does that automatically (and you can use `{foo!r}` to get repr(foo) instead when that is what you need; there is also `{foo!s}` for str(foo) conversion but as I said, this is already the default so you don't need even the `!s` part).